nucleus/__init__.py

```diff
@@ -32,6 +34,8 @@
     "SegmentationAnnotation",
     "SegmentationPrediction",
     "Slice",
+    "UnitTest",
+    "UnitTestMetric",
```

Comment:
Curious what others think of putting all the docs for Nucleus and Model CI in the same location vs a subpage. Before, the docs would be available at `nucleus/modelci`.

Reply:
I like the idea of separating it, I think it's bothersome to have to scroll through all the other classes to have to get to the Model CI client extension

Reply:
Yes thought the same, didn‘t find a quick way and didn‘t have more time.
Already asked @drakejwong for help about this. Will try to finish it off tomorrow.